components/bl-stripe-payment-component/src/helpers/use-card-element.js

```diff
@@ -58,7 +69,9 @@ export function useCardElement(eventHandlers, setIsLoading, setTransactionDetail
       .catch(error => {
         const errorMessage = error.message;
 
-        setErrorMessage(errorMessage);
+        setErrorMessage(
+          errorMessage === ErrorMessages.SERVICE_NOT_AVAILABLE ? ErrorMessages.PLUGIN_NOT_AVAILABLE : errorMessage
```

Comment:
Could we make in a little shorter?

ErrorMessages -> Errors

NOT_AVAILABLE -> UNAVAILABLE